### PR TITLE
zh-trans: update docker-cli-to-kubectl.md

### PR DESCRIPTION
**Pull request by SataQiu**
修正目录结构 1.1.1 -> 1.

### CODE REVIEW

**Comment by xichengliudui** [comment]
/lgtm
/approve
thanks!

**Comment by xichengliudui** [comment]
WHY

File: content/zh/docs/user-guide/docker-cli-to-kubectl.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ approvers:
 title: Docker 用户使用 kubectl 命令指南
 ---
 
-在本文中，我们将向 docker-cli 用户介绍 Kubernetes 命令行如何与 api 进行交互。该命令行工具——kubectl，被设计成 docker-cli 用户所熟悉的样子，但是它们之间又存在一些必要的差异。该文档将向您展示每个 docker 子命令和 kubectl 与其等效的命令。
+在本文中，我们将向 docker-cli 用户介绍 Kubernetes 命令行如何与 api 进行交互。该命令行工具 —— kubectl，被设计成 docker-cli 用户所熟悉的样子，但是它们之间又存在一些必要的差异。该文档将向您展示每个 docker 子命令和 kubectl 与其等效的命令。
```
Comment (idealhack) [inline content/zh/docs/user-guide/docker-cli-to-kubectl.md]: since you are modifying this, why not `——`

**Pull request merged**